lib/messenger/listeners/sqs.rb

```diff
@@ -73,19 +73,12 @@ def receive_messages
 
         def submit_message(message)
           ensure_valid_worker
-
-          # Update this message's visibility so it doesn't expire while we're working on it.
-          @sqs.change_message_visibility({ queue_url:          self.class.config.queue_url,
-                                           receipt_handle:     message.receipt_handle,
-                                           visibility_timeout: self.class.config.visibility_timeout
-                                         })
-
```

Comment:
we don't need to change the visibility since the message timeout is 30 seconds and the only thing we're doing is sending it to Redis.